python/nav/event2.py

```diff
@@ -92,5 +92,5 @@ def end(self, device=None, netbox=None, subid='', varmap=None, alert_type=None):
     def notify(self, device=None, netbox=None, subid='', varmap=None, alert_type=None):
         """Creates and returns a stateless event"""
         event = self.base(device, netbox, subid, varmap, alert_type or self.start_type)
-        event.event_type = event.STATE_STATELESS
+        event.state = event.STATE_STATELESS
```

Comment:
This was a good catch - and I assume this is what was discussed during today's standup as something that needs to be a PR of its own :)